CHANGELOG.md

```diff
@@ -5,7 +5,49 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-* renamed model's `n_classes` to `num_classes`
+
+## [0.7.0] - 2021-09-24
+### Added
+* Initial phase of major usability improvements in `monai.transforms` to support input and backend in PyTorch and NumPy
+* Performance enhancements, with profiling and tuning guides for typical use cases
+* Reproducing training modules and workflows of state-of-the-art Kaggle competition solutions
+* 24 new transforms, including
+  * `OneOf` meta transform
+  * DeepEdit guidance signal transformsfor interactive segmentation
```

Comment:
missing a blank between `transformsfor`.

Reply:
thanks! this is not ready for review yet, I'm still editing it...